public/classroom_poc.html

```
@@ -0,0 +1,733 @@
+        <!-- ---- Chat Sidebar ---- -->
+        <div id="chatPanel" class="w-80 shrink-0 bg-gray-900/70 border-l border-gray-800 flex flex-col">
+            <div class="px-4 py-3 border-b border-gray-800 flex items-center gap-2">
+                <i data-lucide="message-circle" class="w-4 h-4 text-teal-400"></i>
+                <h2 class="text-sm font-semibold text-gray-200">Room Chat</h2>
+            </div>
+            <div id="chatMessages" class="flex-1 overflow-y-auto p-3 space-y-2">
+                <!-- Messages appended here -->
+                <div class="text-center text-xs text-gray-600 py-6">No messages yet!</div>
+            </div>
+            <form id="chatForm" class="p-3 border-t border-gray-800 flex gap-2">
+                <input id="chatInput" type="text" placeholder="Type a message…" autocomplete="off"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)